src/pages/detail-page/sections/pc/MainSection.jsx

```diff
@@ -0,0 +1,149 @@
+import { useRef } from 'react';
+import { motion, useScroll, useTransform } from 'motion/react';
+import useElementHeight from '@hooks/useElementHeight';
+import DonationInfo from '@components/DonationInfo';
+import MainTitle from '../../components/MainTitle';
+import Button from '@components/common/Button';
+import DetailTitle from '@pages/detail-page/components/DetailTitle';
+import DetailContent from '@pages/detail-page/components/DetailContent';
+
+const MainSection = () => {
+  // ref를 통해 제목이 브라우저 가장 바닥에 위치한다. (absolute를 사용하면 제목 아래 본문과 이어지지 않는다.)
+  const [titleRef, top] = useElementHeight();
```

Comment:
`useElementHeight` 훅의 반환값으로 `[ref, height]`로 사용해주셨는데,
여기서는 이름이 통일되지 않아 복잡도가 올라가는 거 같습니다. 변수명을 좀 더 구체적으로 명확하게 사용하는 것이 좋습니다.

```suggestion
  const [titleRef, titleHeight] = useElementHeight();
```